### PR TITLE
Create Scraper for Printer Metrics over SNMP

### PR DESCRIPTION
**Pull request by akshtshrma24**
see #64, for previous. This file adds scraper functionality to quasar. This has hard coded values for now but can be easily changed to make it post real data, by simple changes. This pushes data to influxDB, and creates a database if not available already.

### CODE REVIEW

File: printer/scraperPrinter/influxHandler.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    let bodyData = 'laserJet,tag=CNB9M04409 inkLevel=' + String(snmpData.inkLevel) + '\n';
+    bodyData += 'laserJet,tag=CNB9M04409 macAddy="'  + String(snmpData.macAddy) + '"' + '\n';
+    bodyData+='laserJet,tag=CNB9M04409 currentTonerLevel='+String(snmpData.currentTonerLevel)+'\n';
+    bodyData += 'laserJet,tag=CNB9M04409 memorySize=' + String(snmpData.memorySize) + '\n';
+    bodyData += 'laserJet,tag=CNB9M04409 memoryUsed=' + String(snmpData.memoryUsed) + '\n';
+    bodyData += 'laserJet,tag=CNB9M04409 pagesPrinted=' + String(snmpData.pagesPrinted) + '\n';
+    bodyData+='laserJet,tag=CNB9M04409 serialNumber="'+String(snmpData.serialNumber)+'"'+'\n';
+    bodyData += 'laserJet,tag=CNB9M04409 modelNumber="' + String(snmpData.modelNumber) + '"' + '\n';
+    bodyData += 'laserJet,tag=CNB9M04409 tonerCapacity=' + String(snmpData.tonerCapacity);
+    logger.warn('This was Sent \n' + bodyData);
```
Comment (starphys) [inline printer/scraperPrinter/influxHandler.js]: Make this tag=snmpData.serialNumber to scale to both printers

Comment (akshtshrma24) [reply printer/scraperPrinter/influxHandler.js]: fax

File: printer/scraperPrinter/printerScraper.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    const dataForInflux = await globalThis.influxHandler.formatForInflux(snmpDataFromPrinter);
+    globalThis.influxHandler.writeToInflux(dataForInflux);
+  }
+
+  /**
+  * Starts the Scraper at the desired interval
+  */
+  startScraper() {
+    this.influxHandler.initializeInfluxDb();
+    logger.warn(String(`Starting Query with Interval ${this.intervalSeconds} seconds`));
```
Comment (evanugarte) [inline printer/scraperPrinter/printerScraper.js]:
```suggestion
    logger.info(`Starting Query with Interval ${this.intervalSeconds} seconds`);
```

**Review by evanugarte — APPROVED** [review]
this is the cleanest code we've ever wrote

**Pull request merged**